vignettes/wrapping-apis.Rmd

```diff
@@ -105,21 +106,25 @@ req %>%
   req_dry_run()
 ```
 
+JENNY COMMENT HERE
+
 ### Core request function
 
-Once you've successful made a few requests, it's worth seeing if you can figure out the general pattern so you can wrap it up into a function that will become the core of your package.
+Once you've made a few successful requests, it's worth seeing if you can figure out the general pattern so you can wrap it up into a function that will become the core of your package.
 
 For faker, I spent a little with the [documentation](https://fakerapi.it/en#basic-usage) noting some commonalities:
 
 -   Every URL is of the form [`https://fakerapi.it/api/v1`](https://fakerapi.it/api/v1)`/{resource}`, and data is passed to the resource with query parameters.
     All parameters start with `_`.
 
--   Every resource has three common query parameters: `_locale`, `_quantity`, and `_seed` which have default values of `en_US`, `1` and `null` respectively.
+-   Every resource has three common query parameters: `_locale`, `_quantity`, and `_seed` which have default values of `en_US`, `10`, and `null` respectively.
```

Comment:
So the default for `_quantity` is `10` not `1`? If so, that needs to change in a few other places.

Reply:
The default for the API is 10; but I want to change it to 1 for these demos.